Makefile

```diff
@@ -121,18 +127,21 @@ pinned:
 .PHONY: pinned-dev #CMD Save the current environment for pinned development installation.
 pinned-dev:
 	@echo -e "\n[make pinned-dev] creating file defining pinned conda environment"
-	$(CONDA_ACTIVATE) $(NAME_OR_PREFIX)
+	$(CONDA_ACTIVATE) $(NAME_OR_PREFIX_DEV)
 	conda env export --no-builds | tail -n +2 | head -n -2 > $(FILE_DEV_PINNED)
 	@echo "[make pinned-dev] Pinned environment saved in" $(FILE_DEV_PINNED)
 
 #==============================================================================
 # Run the tests
 #==============================================================================
 
-.PHONY: test    #CMD Run tests.
-test:
-	@echo -e "\n[make test] running all tests"
+.PHONY: tests   #CMD Run tests.
+tests:
```

Comment:
`make test` is more common https://cmake.org/cmake/help/book/mastering-cmake/chapter/Testing%20With%20CMake%20and%20CTest.html

Reply:
Fixed